.github/workflows/deploy.yaml

```diff
@@ -254,6 +278,32 @@ jobs:
           openssl ecparam -name prime256v1 -genkey -noout -out key.pem
           openssl ec -in key.pem -pubout -out key.public.pem
           docker run -i danedmunds/pem-to-jwk:1.2.1 --public --pretty < key.public.pem > key.public.jwk
+          
+      - name: 'Prepare GAIA-X self-description signature request'
```

Comment:
Consider extracing this self-description preparation steps into a composite action to prevent duplication?

Reply:
Yeah I even started it, but then thought the files and attributes are bound to diverge between authorities and companies, and for a spike the current structure is perhaps easier for hackathon participants to work with